book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,34 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
+
+### OVERVIEW
+This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration."
+### PLUGINS PREPERATIONS
+
+#### Add Single Bundle Plugin to project configuration
+Add single bundle plugin and select versions of react-native, react and react-native-zapp-bridge you want to use. Save plugin configuration and rebuild. Please note rebuild is necesery every time you change versions or adding/removing other react native plugins.
```

Comment:
i would put "single bundle plug-in" in quotes

Reply:
Could you provide a bit of explanation what is react-native-zapp bridge? It's important to give external context

Reply:
The React-Native-zapp-bridge package is bundled in automatically ? it cannot be opted out ? what if we don't want to use it ?

Reply:
Yes, it is just because it was like that before we introduce Single Bundle. If we opt out then 2 level side menu will stop working.